main.cpp

```diff
@@ -0,0 +1,34 @@
+#include <iostream>
+
+/*Joe bought 1k stocks for 45.50 , Broker got 2%
+ * Joe sold 1k stocks for 56.90 , Broker got 2%
+ * Output :
+ * Money paid
+ * commission paid
+ * Amount sold for
+ * commission paid
+ * Profit or Loss
+ * Constants - SHARE_BOUGHT , SHARE_SOLD , COMMISSION_RATE , PURCHASE_PRICE , SALE_PRICE
+*/
+
+int SHARE_BOUGHT = 1000 , SHARE_SOLD = 1000 ;
+double COMMISSION_RATE = 0.02, PURCHASE_PRICE = 45.5, SALE_PRICE = 56.9;
+int main() {
+    double sale_bought , sale_sold , sale_profit , commssion_paid_buy , commssion_paid_sale;
+    char new_line = '\n';
+
+    sale_bought = SHARE_BOUGHT * PURCHASE_PRICE;
+    commssion_paid_buy = sale_bought * COMMISSION_RATE;
+
+    std::cout <<"You spent "<<sale_bought<<" on stocks and paid "<<commssion_paid_buy<<" to your broker."<<new_line;
```

Comment:
Interesting use of the new_line variable.
This is fine but it is cheaper to use the literal `'\n'` directly and not save it in a variable.

Really kind of a pedantic request so just take it under advisement.